test/Common/DurableTaskLifeCycleNotificationTest.cs

```diff
@@ -933,7 +933,9 @@ public void OrchestrationEventGridApiConfigureCheck()
         private static Mock<INameResolver> GetNameResolverMock((string Key, string Value)[] settings)
         {
             var mock = new Mock<INameResolver>();
-            foreach (var setting in settings)
+            var settingList = settings.ToList();
+            settingList.Add(("TestTaskHub", string.Empty));
```

Comment:
Why do the lifecycle notification tests need to be updated like this? If it's required to make these tests pass, then I suspect there is some other bug that we're covering up with this change.

Reply:
It was beacuase of the build issues I was facing from the` %{value}%` notation for test that does not use injected `INameResolver`.


Thank you for the comment!